core/modules/filters/parsedate.js

```diff
@@ -0,0 +1,38 @@
+/*\
+title: $:/core/modules/filters/parsedate.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator converting different date formats into TiddlyWiki's date format
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports.parseDate = function(source,operator,options) {
```

Comment:
Filter operator names do not use camel case.

In any case, I'm not sure that the operator is needed – the alternative is to add support to `$tw.utils.parseDate()`, and then the existing date operators will all be able to cope with ISO format.

Reply:
Currently TiddlyWiki's date format is well [defined and documented](https://tiddlywiki.com/#Date%20Fields). Extending parseDate would allow the ISO format to be used everywhere. That can lead to a mix of both date formats in fields. Most of the time this will work. But sometimes plugins or user code, that depends on the date being in the default format will break in hard to debug ways.

In my opinion it would be better to stick with a fixed and well defined date format and use a conversion function.

Reply:
Thanks @FlashSystems I agree.